README.md

```diff
@@ -45,3 +45,21 @@ The following steps should serve as a guide when adding new packages or dependen
 - Add a patch if necessary to `patches` directory.
 - If it's a system dependency, add a build rule to `install.sh`. According to the library, it may be necessary to use `build_library` script or adding a special rule to `build_library_with_toolchain`.
 - That's it! Test your build and watch out build errors. Using `verbose` flags when building should help finding hints about the problems that may arise.
+
+## Building your own apps and libraries on top of ROS cross-compiled workspace
+
+You can use `build_catkin_workspace` as a standalone script to build your Android application or library as a catkin package,
+making use of the ROS cross compiled libraries and system dependencies. The examples under [example_workspace](https://github.com/Intermodalics/ros_android/tree/kinetic/example_workspace/src) can be used as a starting point and as a reference to build your code; in short they are Gradle projects wrapped as Catkin packages.
+
+Here's some guidelines to take into account when creating new projects, using `hello_world` app as an example:
+- Your code should have the structure of a ROS Android package, with a top level `CMakeLists` to make Gradle interact with Catkin. Take [this CMakeLists file](https://github.com/Intermodalics/ros_android/blob/kinetic/example_workspace/src/hello_world_example_app/CMakeLists.txt) as an example.
+- A [package.xml](https://github.com/Intermodalics/ros_android/blob/kinetic/example_workspace/src/hello_world_example_app/package.xml) has to be created, declaring Catkin as the build tool, and the ROS dependencies.
```

Comment:
Mention `rosjava_build_tools` as another required `<build_tool_depend>`?

Reply:
The declaration is actually missing in the examples.

Reply:
Declaration added in https://github.com/Intermodalics/ros_android/pull/101/commits/10a292bad1c021fbe7cb6d9277871cd27ab643d5; I'll add the clarification in the readme as well.